.coveragerc

```diff
@@ -0,0 +1,3 @@
+[run]
+omit =
+    virtualizarr/tests/*
```

Comment:
could we put this in pyproject.toml rather than a new top-level file? e.g., https://github.com/zarr-developers/zarr-python/blob/458299857141a5470ba3956d8a1607f52ac33857/pyproject.toml#L122-L125

Reply:
Could do, but I was trying to avoid having to change deps. We need to change `codecov` to `codecov[toml]`, but let me give that a shot.

Reply:
ok, good call, I was able to eliminate the extra file and put the settings in pyproject.toml by changing the dependency